### ISSUE TITLE
Tooltip für Eingabe Standardisierter Name (Fallback für Issue 185)

### ISSUE THREAD

**Issue by BrittaTSI** [corona-warn-app/cwa-quick-test-frontend#177]
Um die Nutzer des Schnelltestportals bei der Eingabe des korrekten standardisierten Namen unterstützen, muß ein Tooltip erstellt werden, in dem erklärt wird, wo der Tester die notwendigen Informationen findet.

1. - [ ] Formulierung Text für den Tooltip (Scoping)
2. - [ ] Implementierung Tooltip (Gordon)
 
ad 1 - der standardisierte Name findet sich z.B. im maschinenlesbaren Bereich des deutschen Personalausweises. 
TBD - wie wird der SN bei Nicht-Deutschen / Nicht-EU-Bürgern ermittelt?

**Comment by dideldei** [comment]
Wozu benötigt man ihn? Irgednwie konterkariert das doch langsam die gesamte Datensparsamkeit, was im Schnelltestportal als Pflichtfelder verlangt wird. Das ist mehr als ich an das Gesundheitsamt melde, wenn ich einen positiven PCR-Test habe. Kickt doch mal eure Juristen aus dem Projekt.

**Comment by Ein-Tim** [comment]
@dideldei 

If you want to get your concerns addressed I suggest to open an issue in the documentation repository.
I don't understand how a tooltip would help with getting more data, it only explains where one can find the required infos.

**Comment by dideldei** [comment]
You are right. It is the wrong location. But it is rather frustrating to see where this meaningfull project is going. If we did not used the "standardisierter Name" we would not need a tooltip. I think it is about to lose track to create sth helpful. It is becoming a UX nightmare and a lawyers dream.

**Comment by Ein-Tim** [comment]
Okay, I don't have access to a quick test portal so I can't tell. If you found UX issues I suggest to open a new issue here.

**Comment by BrittaTSI** [comment]
@dideldei @Ein-Tim 
Thanks for your input.
We are talking about two different topics here:

1. The actual quicktest process. The Schnelltestportal interacts with the Corona-Warn-App (see 2. below) but it is a completely separate system.  If you want to do a so-called Bürgertest at a Point of Care you are required by law to provide a minimum set of personal data - Vorname Nachname, Date of Birth, address, phone number... that is needed e.g. to contact you to compile a test certificate, deliver the result and to inform the Gesundheitsamt if your test result is positive. Some PoC have their own systems to collect this data and issue test results, some use the Schnelltestportal the CWA-team is building for BMG/RKI.
We are not collecting **additional** data here but just adding a different **encoding** of the pre-existing fields Vorname and Nachname. "Standardisierter Vorname/Nachname" are mandatory fields in the specification for the EU Digital COVID Certificates we are currently integrating into the CWA ecosystem. AfaIk they are meant to ensure interoperability between covid test certificates from different countries as names can contain language-specific letters like umlauts, ß, accents etc. that could cause problems with systems that use other languages. (For example we have already collected the values "Renée Schüßler" and just add the encoding "Renee Schuessler".) 
The tooltip is just meant to help the people entering the data into the Schnelltestportal to transcribe the patient's name into the standardized format.
2. The additional processing of the test result in the Corona-Warn-App is always optional. The patient that visits a PoC to do a test can choose to get their test result only as a printout or via eMail / download. If they want to receive it additionally in their Corona-Warn-App they can choose to get a certificate that meets the EU requirements or simply the result without any personal data. In that case the PoC will generate a QR-code (e.g. using the Schnelltestportal) that the patient scans with their CWA to get the test result. 

Long story short - there is still the possibility to import your test result into the CWA without any further personal data. 
For the actual quicktest, however, you will always have to provide personal data. The Standardisierter Vor/Nachname are not additional data but just different encodings to ensure European interoperability.

**Comment by vaubaehn** [comment]
Dear @BrittaTSI , please see https://github.com/corona-warn-app/cwa-quick-test-frontend/issues/185#issuecomment-875038144

**Comment by BrittaTSI** [comment]
Prefarably https://github.com/corona-warn-app/cwa-quick-test-frontend/issues/185 should be implemented so the standardised name is generated automatically and the tool tip will no longer be necessary.

**Comment by BrittaTSI** [comment]
Obsolete as soon as https://github.com/corona-warn-app/cwa-quick-test-frontend/issues/185 is implemented
